tests/integration/repo_test.go

```diff
@@ -256,3 +256,23 @@ func TestViewRepoDirectory(t *testing.T) {
 	assert.Zero(t, repoTopics.Length())
 	assert.Zero(t, repoSummary.Length())
 }
+
+func TestMarkDownImage(t *testing.T) {
+	defer tests.PrepareTestEnv(t)()
+
+	session := loginUser(t, "user2")
+
+	req := NewRequest(t, "GET", "/user2/repo1/src/branch/home-md-img-check")
+	resp := session.MakeRequest(t, req, http.StatusOK)
+
+	htmlDoc := NewHTMLParser(t, resp.Body)
+	_, exists := htmlDoc.doc.Find("img[src=\"/user2/repo1/media/branch/home-md-img-check/test-fake-img.jpg\"]").Attr("src")
```

Comment:
```suggestion
	_, exists := htmlDoc.doc.Find(`img[src="/user2/repo1/media/branch/home-md-img-check/test-fake-img.jpg"]`).Attr("src")
```

Reply:
In some places, we are using the same format.
Such as: 
https://github.com/go-gitea/gitea/blob/ffce336f1802b2f2298fc8fd27e815086702c812/tests/integration/repo_generate_test.go#L39
Is it necessary to fix them all in another PR?